docs/bulk_data.rst

```diff
@@ -105,6 +96,54 @@ is used to support the ``ParentId`` self-lookup on ``Account``.
 CumulusCI defaults to using the Bulk API in Parallel mode. If required to avoid row locks,
 specify the key ``bulk_mode: Serial`` in each step requiring the use of serial mode.
 
+Database Mapping
+----------------
+
+CumulusCI's definition format includes considerable flexibility for use cases where datasets
+are stored in SQL databases whose structure is not identical to the Salesforce database.
+Salesforce objects may be assigned to arbitrary database tables, and Salesforce field names
+mapped to arbitrary columns.
+
+For new mappings, it's recommended to allow CumulusCI to use sensible defaults by specifying
+only the Salesforce entities. Existing datasets are likely to include explicit database mappings,
```

Comment:
Shall we update the mapping generator?

Reply:
Yes, but. The list format is not good for projects that need to be able to change the Salesforce API name mappings (almost all of ours) to utilize both namespaced and non-namespaced orgs. I'd like to revise the mapping generator post-namespace inference so that it can take advantage of that feature too.